projects/npm-tools/packages/changelog-generator/src/command.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const {exec} = require('child_process');
+const {error} = require('./console');
+
+function runCommand(command, ...args) {
+	exec(command, (err, stdout, stderr) => {
```

Comment:
you will need to use `spawnSync` here because otherwise the script won't wait for the `prettier` process to finish